pointcloud/pointcloud_utils.go

```diff
@@ -24,10 +24,7 @@ func BoundingBoxFromPointCloudWithLabel(cloud PointCloud, label string) (spatial
 	// calculate extents of point cloud
 	meta := cloud.MetaData()
 	dims := r3.Vector{math.Abs(meta.MaxX - meta.MinX), math.Abs(meta.MaxY - meta.MinY), math.Abs(meta.MaxZ - meta.MinZ)}
-
-	// calculate the spatial average center of a given point cloud
-	n := float64(cloud.Size())
-	mean := r3.Vector{meta.TotalX() / n, meta.TotalY() / n, meta.TotalZ() / n}
+	mean := r3.Vector{dims.X / 2, dims.Y / 2, dims.Z / 2}
```

Comment:
you shouldn't use dims, as that is an the absolute value of the distance between the max and min. What you are looking for is the point inbetween the max and min, which would be 

`center := r3.Vector{(meta.MaxX + meta.MinX)/2.0, (meta.MaxY + meta.MinY)/2.0, (meta.MaxZ + meta.MinZ)/2.0}`

Reply:
I just put this up for @micheal-parks to discuss with you, but happy to make that change if we think that's more accurate!